scripts/superdesk-monitoring/monitoring.js

```diff
@@ -253,6 +254,19 @@
             scope: {
                 type: '=',
                 state: '='
+            },
+            link: function(scope, elem) {
+                var containerElem = elem.find('.content-list');
+                containerElem.on('scroll', handleContainerScroll);
```

Comment:
i think you should call `containerElem.off('scroll');` on `$destroy` event, otherwise this will leak memory